snakemake_conda/Snakefile

```diff
@@ -0,0 +1,7 @@
+rule run_python_script:
```

Comment:
Make the rule name descriptive of what is being achieved by the script.

Reply:
Added a descriptive new rule in the same Snakefile.